pom.xml

```diff
@@ -184,7 +185,19 @@
           <xmlOutput>true</xmlOutput>
           <effort>Max</effort>
           <threshold>Low</threshold>
-          <excludeFilterFile>${findbugs.config}</excludeFilterFile>
+          <excludeFilterFile>${project.build.directory}/plugin-config/findbugs/hotels-findbugs-exclude.xml</excludeFilterFile>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>com.github.spotbugs</groupId>
+        <artifactId>spotbugs-maven-plugin</artifactId>
+        <version>${spotbugs.maven.plugin.version}</version>
+        <configuration>
+          <encoding>${project.build.sourceEncoding}</encoding>
+          <xmlOutput>true</xmlOutput>
+          <effort>Max</effort>
+          <threshold>Low</threshold>
+          <excludeFilterFile>${project.build.directory}/plugin-config/findbugs/hotels-findbugs-exclude.xml</excludeFilterFile>
```

Comment:
Let's have different config files for the different plugins so we can evolve them separately.